projects/examples/src/components/spotlight-search/spotlight-search-example.component.ts

```diff
@@ -24,53 +22,44 @@ export class SpotlightSearchExampleComponent implements OnInit, OnDestroy {
     kbdShortcut = 'mod+f';
     spotlightOpen: boolean;
     private registrationId: string;
+    private readonly mousetrap: MousetrapInstance;
 
     constructor(
         private fb: FormBuilder,
         private actionsSearchProvider: ActionsSearchProvider,
-        private spotlightSearchService: SpotlightSearchService,
-        private changeDetectorRef: ChangeDetectorRef
+        private spotlightSearchService: SpotlightSearchService
     ) {
+        // Create an instance of mouse trap within the constructor so that any bount shortcut event handler
+        // would be executed within the angular zone
+        this.mousetrap = new Mousetrap();
+        // For this example we'd like mouse trap to rum always
```

Comment:
nit: " For this example, we'd like mouse trap to always run".

 Doesn't that mean we should have an API to do this?